lib/trento/application/event_handlers/alerts_event_handler.ex

```diff
@@ -7,14 +7,15 @@ defmodule Trento.AlertsEventHandler do
     application: Trento.Commanded,
     name: "alerts_event_handler"
 
-  alias Trento.Domain.Events.ClusterHealthChanged
   alias Trento.Hosts.Events.HostHealthChanged
 
   alias Trento.SapSystems.Events.{
     DatabaseHealthChanged,
     SapSystemHealthChanged
   }
 
+  alias Trento.Clusters.Events.ClusterHealthChanged
```

Comment:
I wouldn've put this before the `Hosts` events, as it is alphabetical order, but just OCD XD
This applies to all files by the way